event/src/main/java/com/bazaarvoice/emodb/event/db/astyanax/AstyanaxEventReaderDAO.java

```diff
@@ -340,17 +340,20 @@ public void readNewer(String channel, EventSink sink) {
      */
     private Iterator<Column<ByteBuffer>> readManifestForChannel(final String channel, final boolean weak) {
         final ByteBuffer oldestSlab = weak ? _oldestSlab.getIfPresent(channel) : null;
-        final ConsistencyLevel consistency;
+        ConsistencyLevel consistency = ConsistencyLevel.CL_LOCAL_ONE;
 
         RangeBuilder range = new RangeBuilder().setLimit(50);
-
         if (oldestSlab != null) {
             range.setStart(oldestSlab);
-              consistency = ConsistencyLevel.CL_LOCAL_ONE;
+            _log.info("Consistency level set to ", consistency);
         } else {
-              consistency = ConsistencyLevel.CL_LOCAL_QUORUM;
+            try {
+                consistency = ConsistencyLevel.CL_TWO;
```

Comment:
are you not reading from env variable property?

Reply:
No, we are passing it from the file itself as it was in the production earlier.

Reply:
It was suggested to pass it as an environment variable before and you have paused the activity to implement the same? why is it going again in this way?

Reply:
We tested with passing environment variable, no issues with that testing but that environment variable was also added to tools page when connecting to instances. So passed it through file itself.
Will pass the environment variable from ops and deploy. I will let you know once tested.
![Screenshot 2024-06-13 at 6 59 10 AM](https://github.com/bazaarvoice/emodb/assets/134580247/7fe80d14-f4a6-4ad5-9fa5-af7261274ae1)